### PR TITLE
Run linters and black on all python files

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

#39 

### Description of the Change

Add linting for all python files

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -38,14 +40,16 @@ def start():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def start_server(request):
+def start_server(
+    request,
+):  # Input must exist due to 'fixture'. pylint:disable=unused-argument
```
Comment (magnusbaeck) [inline tests/conftest.py:45]: Side note: Consider installing [pylint-pytest](https://pypi.org/project/pylint-pytest/) (not to be confused with pytest-pylint ;-)) to avoid having to suppress this kind of pylint violation.

File: requirements.txt [hunk 2]
```diff
@@ -15,7 +15,7 @@
 # numpy==1.13.3
 # scipy==1.0
 #
-eiffellib==1.1.0
+eiffellib==1.1.1
```
Comment (magnusbaeck) [inline requirements.txt:18]: This probably belongs in a separate commit. Besides, isn't this file autogenerated or something? At the very least the `install_requires` directive in setup.cfg should be updated in tandem?

Comment (magnusbaeck) [reply requirements.txt:18]: Indeed, just changing requirements.txt doesn't affect what ends up in the dependencies of the package that you build from setup.py. I've verified that 1.1.1 solves my consumption problem so I'll send a PR.

File: tests/conftest.py (hunk 1 above)
Comment (t-persson) [reply tests/conftest.py:45]: Let's do that in a separate commit as I have a lot to do with ETOS now :)
